### ISSUE TITLE
Please update to SnapKit 4.0.0

### ISSUE THREAD

**Issue by halfwaychair** [Daltron/NotificationBanner#51]
![screen shot 2017-09-19 at 10 49 08 pm](https://user-images.githubusercontent.com/31868672/30624683-dbccf9ea-9d8c-11e7-8def-1147220828a5.png)

**Comment by Daltron** [comment]
Hi @halfwaychair! 😄 Can you provide a reason please?

**Comment by YuriSolodkin** [comment]
I think the reason is that it does not compile with Swift 4

**Comment by Chiefinho** [comment]
Hi, I am in the same situation. Including NotificationBanner in a swift 4 project means it will not compile as SnapKit 3.2 is not compatible with swift 4

**Comment by Daltron** [comment]
@YuriSolodkin @Chiefinho Swift 3 and Swift 4 work interchangeably. I think the problem is you are compiling all your pods to use Swift 4. You need to tell your pod file which ones shouldn't be compiled into Swift 4. As a temporary fix, you can do this and NotificationBanner will not be compiled into swift 4

This can go at the bottom of your Podfile
```ruby
post_install do |installer|
    installer.pods_project.targets.each do |target|
        if target.name == 'NotificationBannerSwift'
            target.build_configurations.each do |config|
                config.build_settings['SWIFT_VERSION'] = '3.2'
            end
        end
    end
end
```

**Comment by Daltron** [comment]
Please feel free to open up a pull request with Swift 4 compatibility if you need it right away! 😄 It might be a few days before I can get to this.

**Comment by Chiefinho** [comment]
Thanks, that helped a lot. I did need to add:
```ruby
if target.name == 'SnapKit'
    target.build_configurations.each do |config|
        config.build_settings['SWIFT_VERSION'] = '3.2'
    end
end
```
to get it to compile

**Comment by Daltron** [comment]
@Chiefinho glad to hear! I will try to get everything ported over to Swift 4 when I have more time.

**Comment by halfwaychair** [comment]
Compiled, thank you

**Comment by Daltron** [comment]
@FunProject Swift 4 support is now available in [v1.5.0](https://github.com/Daltron/NotificationBanner/releases/tag/1.5.0) release! 🎉 You should no longer need to have the post_installer method run for NotificationBanner. If you have other pods that require the swift version to be 3.2, it will still need to remain in place.
